hack/release-automater/release_overview.py

```
@@ -0,0 +1,369 @@
+
+def get_lifecycle_data():
+    url = "https://access.redhat.com/product-life-cycles/api/v1/products"
+    params = {
+        "name": "Openshift Container Platform"
+    }
+    headers = {
+        "User-Agent": "Mozilla/5.0",
+        "Accept": "application/json"
+    }
+
+    res = requests.get(url, headers=headers, params=params)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)